openhands/events/observation/commands.py

```diff
@@ -13,6 +13,7 @@ class CmdOutputObservation(Observation):
     exit_code: int = 0
     hidden: bool = False
     observation: str = ObservationType.RUN
+    meta: str = ''
```

Comment:
is there a better name for this?

Reply:
Perhaps `location`, or `loc`, or `current_dir`?

Reply:
I renamed it to `interpreter_details`